plugins/woocommerce-admin/client/customize-store/style.scss

```diff
@@ -312,3 +312,14 @@ body.woocommerce-customize-store.js.is-fullscreen-mode {
 	}
 }
 
+.woocommerce-cys-wordpress-header-logo {
+	position: absolute;
+	top: 28px;
+	left: 24px;
+}
+
+.edit-site-layout__hub svg.woocommerce-cys-wordpress-header-logo {
+	width: 24px !important;
+	height: 24px !important;
+}
```

Comment:
Can't we avoid to use `!important`?

Reply:
Okay interesting. After the rebase, I didn't need to use `!important` anymore.